packages/neuron-wallet/src/services/wallets.ts

```diff
@@ -39,13 +39,18 @@ export default class WalletService {
     this.setActive(walletStore.getAllWallets()[0].id)
   }
 
-  public index = (): Wallet[] => {
-    return walletStore.getAllWallets().map(w => ({
-      id: w.id,
-      name: w.name,
-      address: `address of ${w.id}`,
-      publicKey: Buffer.from(`address of ${w.id}`),
-    }))
+  public index = (
```

Comment:
Discussion: since `neuron-wallet` is not UI layer or web service, `index` sounds very odd. Would `list` or something else be a better name even we think this as RESTful resource?

Reply:
Agree, @classicalliu do you have any idea about the method names?

Reply:
`getWallets` ?

Reply:
The name of the methods could be determined later.